### PR TITLE
Rciti2 - test deploy

### PR DESCRIPTION
**Pull request by asiripanich**
- Several updates from master 
- added a second on boarding screen to explain and guide the user to setup the location permission for emTripLog of their phone 
- Updated survey forms
- Justified text in the consent page
- Cleaned up older versions of the surveys

### CODE REVIEW

File: www/i18n/en.json [hunk 1]
```diff
@@ -245,17 +245,28 @@
         }
     },
 
+    "intro": {
+        "permissions": {
+            "locationPermExplanation-android-lt-6": "you accepted the permission during installation. You don't need to do anything now.",
+            "locationPermExplanation-android-gte-6": "please select 'allow'",
+            "locationPermExplanation-ios-lt-13": "please select 'Always allow'. This allows us to protect you even when you are not actively using the app",
```
Comment (shankari) [inline www/i18n/en.json:252]: Didn't you want to change "protect" here?

File: www/js/survey/enketo-survey.js [hunk 2]
```diff
@@ -100,7 +100,7 @@ angular.module('emission.survey.enketo.launch', [
 
   function initConfirmSurvey(opts) {
     return initSurvey({
-      form_location: 'json/20200420-01-trip-end-survey.json',
+      form_location: 'json/trip-end-survey.json',
```
Comment (shankari) [inline www/js/survey/enketo-survey.js:103]: Is this what you want? I thought you wanted the most recent version of the survey...

File: www/i18n/en.json (hunk 1 above)
Comment (asiripanich) [reply www/i18n/en.json:252]: 'protect' is actually not a bad idea for now.. :)

File: www/js/survey/enketo-survey.js (hunk 2 above)
Comment (asiripanich) [reply www/js/survey/enketo-survey.js:103]: That's the most recent version. I removed all older forms.

**Comment by shankari** [comment]
I'm approving this for now since the changes are relatively minor, but in the future, to make it easier to review, it would be good to split to merge from master (no review needed) and the changes that you made.

**Comment by asiripanich** [comment]
> approving this for now since the changes are relatively minor, but in the future, to make it easier to review, it would be good to split to merge from master (no review needed) and the changes that you made.

Sorry for the mess. :( Will do better next time.

**Comment by shankari** [comment]
no worries, as you probably saw, I deployed the channel already

**Pull request merged**